rotkehlchen/db/dbhandler.py

```diff
@@ -3421,3 +3421,25 @@ def delete_user_note(self, identifier: int) -> None:
             write_cursor.execute('DELETE FROM user_notes WHERE identifier=?', (identifier,))
             if write_cursor.rowcount == 0:
                 raise InputError(f'User note with identifier {identifier} not found in database')
+
+    def get_nft_mappings(self, identifiers: List[str]) -> Dict[str, dict]:
+        """
+        Given a list of nft identifiers, return a list of nft info (id, name, collection_name)
+        for those identifiers.
+        """
+        result = {}
+        identifiers_query = f'identifier IN ({",".join("?" * len(identifiers))})'
+        with self.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT identifier, name, collection_name FROM nfts WHERE ' + identifiers_query,
+                identifiers,
+            )
+            for entry in cursor:
+                result[entry[0]] = {
+                    'name': entry[1],
+                    'symbol': None,  # for compliance with assets mappings from globaldb
+                    'asset_type': 'nft',
```

Comment:
If the type is `NFT` we might be omitting this field. We should ask @lukicenturi about what is better here